### ISSUE TITLE
City Mapa-thon - (San) Juan more time

### ISSUE THREAD

**Issue by govvin** [mapamore/events#7]
RSVP and registration: https://ti.to/mapamore/juan-more-time
Objective: Completing pending and incomplete DRR facilities, basic data for barangays in San Juan city.

See: https://github.com/mapamore/events/issues/5

**Comment by govvin** [comment]
Thank you to all the participants who spent their Saturday afternoon with us, mapping the city of San Juan, over Independence day weekend. We're also thankful to San Juan City (and their DRRMO) for making the venue and facilities available. 

And a shout out to Mapillary, for the refreshments.

This is the second city  mapa-thon, and a follow-up activity to enhance the Free/Open data collection of the city of San Juan, and to complete the leftover tasks from the last mapa-thon, chiefly working in the second district of the city:
>![image](https://user-images.githubusercontent.com/5821688/27078326-e52b0168-5065-11e7-93b0-75991d2c3be2.png)
>Data © OpenStreetMap contributors (ODbL); Image © MapCraft

>![image](https://user-images.githubusercontent.com/5821688/27062722-89ed06a2-501f-11e7-89d3-693065148ce1.png)
>Data © OpenStreetMap contributors (ODbL); Image © WHODIDIT

## Highlights

### Evacuation Shelters and `emergency:` prefixes
The designated evacuation shelters in the city are re-purposed facilities made for a different, primary purpose.  Hence, we adopted a tagging convention initiated by the [OSM-tw][^10] community, and now widely used in Taiwan, where the prefix tag `emergency:` is added to the secondary tags related to emergency and DRR mapping.

For example, a sports centre that's been designated by the village council as an evacuation shelter for displaced persons, may then be tagged, as appropriate, as follows,:

```
leisure=sports_centre
sport=multi
name=Palakasan Covered Court
emergency:social_facility=shelter
emergency:social_facility:for=displaced
emergency:social_facility:capacity=*
emergency:shelter_type=typhoon
```

During non-emergencies, they function as originally intended - a sports center, but are re-purposed as shelters during emergencies. Would they have been dedicated shelters, the `emergency:` prefix are dropped and  the are simply tagged as:

```
name=Palakasan Evacuation Shelter
social_facility=shelter
social_facility:for=displaced
social_facility:capacity=*
shelter_type=typhoon
```

>[![image](https://user-images.githubusercontent.com/5821688/27076793-c1c83628-5060-11e7-8f80-e3f78e2e0e9d.png)](http://overpass-turbo.eu/s/pIq)
>We've completely mapped the evacuation centers, barangay halls of the city, except for one barangay where no data was available.
> Click to view data and details on [Overpass Turbo](http://overpass-turbo.eu/s/pIq). © OpenStreetMap contributors.

The DRRMO did not provide the capacity data of the facilities, and that would've been useful to have. 

### Fire Stations and Hydrants
>
![image](https://user-images.githubusercontent.com/5821688/27063428-4d59326a-5024-11e7-874f-a1fb727b140f.png)
>Participants from the Bureau of Fire Protection (their back to the camera): Inspector Antonio (right), and Officer Aquino, listen intently to Mr. Rally de Leon's (facing the camera) lightning talk, during the break.

The Bureau of Fire Protection (BFP) is the country's national agency (under the Department of Interior) responsible for implementing national policies related to fire fighting and prevention. Two personnel from their local office participated in the mapa-thon. They expressed their plans to pursue the validation and mapping of the city's fire-fighting resources.

>![image](https://user-images.githubusercontent.com/5821688/27063176-af67be9c-5022-11e7-8ecc-a71c415ca407.png)
> Copy of the Fire Hydrant Map of San Juan City (by the Maynilad Corporation) brought by the BFP crew.

### ToFix tasks and Validation
> ![image](https://user-images.githubusercontent.com/5821688/27078424-3e69a298-5066-11e7-82b5-60f4146d96f6.png)
> Advance mappers use JOSM to work on [To-Fix task #13](https://github.com/OSMPH/papercut_fix/issues/13)

>![image](https://user-images.githubusercontent.com/5821688/27078520-7d273f5e-5066-11e7-8b88-5e8e18aa1690.png)
> Eugene helping with the validation of the participants' edits.

### Blitz Decks & Pizza
To add some fun and variation to our mapa-thon, we introduced a longer break period coupled with Blitz Decks.  The floor was made available to anyone who'd like to present about anything (hopefully, Free/Open Source or Geo-related, but any topics is welcome (except religion, and politics).i

#### OSM and Project NOAH
>![image](https://user-images.githubusercontent.com/5821688/27076533-0c86b712-5060-11e7-911e-3f2c5e1c09bd.png)
>Here's Dianne, getting ready for her talk about how Project NOAH uses OSM - fueled by pizza and soda, courtesy of our friends from [Mapillary](https://mapillary.com/app) - yay!

#### Neighborhood Mapping and Addressing
>![image](https://user-images.githubusercontent.com/5821688/27076596-45e82b6c-5060-11e7-8fd9-1436f8152242.png)
>Rally de Leon talks about his experience in neighborhood mapping and the importance of addressing.

#### Hail Hydra(nt)!
>![image](https://user-images.githubusercontent.com/5821688/27077513-dd27e0b0-5062-11e7-888f-c7caad65a8d9.png)
> Mapping Hydrants in OSM, and how we got our first hydrants, and how El Deposito figured in our history, by yours truly.

### Conclusion
All these data sets are now available from [OpenStreetMap](https://openstreetmap.org). Thank you to everyone who made this possible. We're looking forward to our next mapa-thon!

Got questions, or ideas in mind? Send us a [Telegram](https://t.me/mapamore), an email mapamore AT ngnuity DOT xyz, or follow us on [Twitter](https://twitter.com/MapAmorePH).

I enjoyed hearing the different viewpoints coming from people who came to try out mapping for the first time:
>
![image](https://user-images.githubusercontent.com/5821688/27077675-73e7125a-5063-11e7-98f9-f697989e2a35.png)
> Kids to remind us to have fun while we're working. :)

Again, we failed to take a proper group photo. :crying_cat_face:  Everyone's too busy mapping, or helping out.

[^10]: https://osmtw.hackpad.com/OSM-lis34MoHRDu "OSMtw Emergency prefix tags"

**Comment by crossonrose** [comment]
Reopen!
